### PR TITLE
Sitvanit/command to example

### PR DESCRIPTION
**Pull request by sitvanit83**
Added files for:
link according to cvl features.

### CODE REVIEW

File: CVLByExample/README.md [hunk 1]
```diff
@@ -1,177 +1,49 @@
...
+| **assert**        | [assert](https://github.com/Certora/Examples/blob/master/DEFI/ConstantProductPool/certora/spec/ConstantProductPool.spec#L75) |
+| **assert_uint256** | [assert_uint256](https://github.com/Certora/Examples/blob/master/DEFI/ERC20/certora/specs/ERC20Fixed.spec#L180) |
+| **at**            | [at](https://github.com/Certora/Examples/blob/master/CVLByExample/Storage/certora/specs/storage.spec#L113) |
+| **builtin rule**  | [builtin rule](https://github.com/Certora/Examples/blob/master/CVLByExample/Reentrancy/certora/spec/ReadOnlyReentrancy.spec#L1) |
+| **calldataarg**   | [calldataarg](https://github.com/Certora/Examples/blob/master/DEFI/ERC20/certora/specs/ERC20Fixed.spec#L83) |
+| **cvldoc @param** | [cvldoc @param](https://github.com/Certora/Examples/blob/master/CVLByExample/Structs/BankAccounts/certora/specs/structs.spec#L44) |
+| **cvldoc @title** | [cvldoc @title](https://github.com/Certora/Examples/blob/master/CVLByExample/NativeBalances/certora/specs/Auction.spec#L11) |
```
Comment (nd-certora) [inline CVLByExample/README.md:12]: put cvldoc in one category

File: CVLByExample/README.md [hunk 2]
```diff
@@ -1,177 +1,49 @@
...
+| **CVL function**  | [CVL function](https://github.com/Certora/Examples/blob/master/DEFI/LiquidityPool/certora/specs/pool.spec#L24), [`override`](https://github.com/Certora/Examples/blob/master/CVLByExample/Import/certora/specs/sub.spec#L38) |
+| **definition**    | [`definition`](https://github.com/Certora/Examples/blob/master/CVLByExample/Import/certora/specs/base.spec#L22) |
+| **envfree**       | [`envfree`](https://github.com/Certora/Examples/blob/master/DEFI/ERC20/certora/specs/ERC20Fixed.spec#L8) |
```
Comment (nd-certora) [inline CVLByExample/README.md:15]: envfree can go into methods block

File: CVLByExample/README.md [hunk 3]
```diff
@@ -1,177 +1,49 @@
...
+| **optional**      | [`optional`](https://github.com/Certora/Examples/blob/master/CVLByExample/Optional/certora/specs/Base.spec#L5) |
+| **expect**        | ['expect'](https://github.com/Certora/Examples/blob/master/CVLByExample/Summarization/MultiContract/certora/specs/FunctionSummary.spec#L11) |
```
Comment (nd-certora) [inline CVLByExample/README.md:28]: expect and optional can go into methods block

File: CVLByExample/README.md [hunk 4]
```diff
@@ -1,177 +1,49 @@
...
+| **CVL function**  | [CVL function](https://github.com/Certora/Examples/blob/master/DEFI/LiquidityPool/certora/specs/pool.spec#L24), [`override`](https://github.com/Certora/Examples/blob/master/CVLByExample/Import/certora/specs/sub.spec#L38) |
+| **definition**    | [`definition`](https://github.com/Certora/Examples/blob/master/CVLByExample/Import/certora/specs/base.spec#L22) |
+| **envfree**       | [`envfree`](https://github.com/Certora/Examples/blob/master/DEFI/ERC20/certora/specs/ERC20Fixed.spec#L8) |
+| **expect**        | [`expect`](https://github.com/Certora/Examples/blob/master/CVLByExample/Summarization/MultiContract/certora/specs/FunctionSummary.spec#L11) |
+| **filtered**      | [`filtered`](https://github.com/Certora/Examples/blob/master/CVLByExample/Reentrancy/certora/spec/Reentrancy.spec#L29) |
+| **forall**        | [`forall`](https://github.com/Certora/Examples/blob/master/CVLByExample/QuantifierExamples/DoublyLinkedList/certora/spec/dll-linkedcorrectly.spec#L13) |
+| **Function call** | [Function call](https://github.com/Certora/Examples/blob/master/DEFI/ERC20/certora/specs/ERC20Fixed.spec#L123) |
```
Comment (nd-certora) [inline CVLByExample/README.md:19]: every thing else is lower case so also here, maybe only CVL has to be with uppercase

File: CVLByExample/README.md [hunk 5]
```diff
@@ -1,177 +1,49 @@
...
+| **hook**          | [`Sstore`](https://github.com/Certora/Examples/blob/master/DEFI/ERC20/certora/specs/ERC20Full.spec#L88), [`Sload`](https://github.com/Certora/Examples/blob/master/DEFI/ERC20/certora/specs/ERC20Full.spec#L84) |
+| **import**        | [`import`](https://github.com/Certora/Examples/blob/master/CVLByExample/Import/certora/specs/sub.spec#L1) |
+| **invariant**     | [simple](https://github.com/Certora/Examples/blob/master/CVLByExample/Invariant/certora/specs/BallGame.spec#L7), [strengthening](https://github.com/Certora/Examples/blob/master/CVLByExample/Invariant/certora/specs/BallGameCorrect.spec#L7), [`preserved with (env e)`](https://github.com/Certora/Examples/blob/master/DEFI/ERC20/certora/specs/ERC20Broken.spec#L128), `requireInvariant` [in a rule](https://github.com/Certora/Examples/blob/master/DEFI/ConstantProductPool/certora/spec/ConstantProductPool.spec#L190), [in a `CVL` function](https://github.com/Certora/Examples/blob/master/DEFI/LiquidityPool/certora/specs/pool.spec#L27) |
```
Comment (nd-certora) [inline CVLByExample/README.md:23]: can we make it look better:
image

maybe line break or inner table?

File: CVLByExample/README.md [hunk 6]
```diff
@@ -1,177 +1,49 @@
...
...Pool.spec#L36), [`NONDET`](https://github.com/Certora/Examples/blob/master/CVLByExample/Summarization/Keywords/certora/specs/NondetVsHavoc.spec#L2), [`HAVOC_ALL`](https://github.com/Certora/Examples/blob/master/CVLByExample/Summarization/Keywords/certora/specs/NondetVsHavoc.spec#L3), Summary Application [ALL](https://github.com/Certora/Examples/blob/master/CVLByExample/Summarization/MultiContract/certora/specs/InternalExternalSummary.spec#L14), [UNRESOLVED](https://github.com/Certora/Examples/blob/master/CVLByExample/Summarization/Library/DirectSummary/certora/specs/AllDirect.spec#L12), [direct summarization](https://github.com/Certora/Examples/blob/master/CVLByExample/Summarization/Library/DirectSummary/certora/specs/AllDirect.spec#L9), [wildcard summarization](https://github.com/Certora/Examples/blob/master/CVLByExample/Summarization/Library/DirectSummary/certora/specs/AllDirect.spec#L12), [CVL Function Summary](https://github.com/Certora/Examples/blob/master/CVLByExample/Summarization/WithEnv/WithEnvCVLFunctionSummary/withEnvSummary.spec#L7), [Ghost Summary](https://github.com/Certora/Examples/blob/master/CVLByExample/Summarization/WithEnv/WithEnvGhostSummary/WithEnv.spec#L10) |
```
Comment (nd-certora) [inline CVLByExample/README.md:26]: also here, can it look better?

**Review by nd-certora — APPROVED** [review]
a few suggestions

File: CVLByExample/README.md (hunk 1 above)
Comment (liav-certora) [reply CVLByExample/README.md:12]: done

File: CVLByExample/README.md (hunk 2 above)
Comment (liav-certora) [reply CVLByExample/README.md:15]: done.

File: CVLByExample/README.md (hunk 3 above)
Comment (liav-certora) [reply CVLByExample/README.md:28]: done.

File: CVLByExample/README.md (hunk 5 above)
Comment (liav-certora) [reply CVLByExample/README.md:23]: done

File: CVLByExample/README.md (hunk 6 above)
Comment (liav-certora) [reply CVLByExample/README.md:26]: yes

File: CVLByExample/README.md (hunk 4 above)
Comment (liav-certora) [reply CVLByExample/README.md:19]: Actually it does make sense to keep Function with upper case. A function call is not something unique to CVL so function in here is more of a work than a keyword in CVL. In a table the indexes should be upper cased as they are used as titles unless it is a keyword of some sorts (like 'filtered' for examples). However here 'Function' isn't referring to the keyword 'function' but to the act of function calling.

**Pull request merged**